connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/outbound/OutboundConnectorRuntimeConfiguration.java

```diff
@@ -41,12 +44,16 @@ public OutboundConnectorManager outboundConnectorManager(
       OutboundConnectorFactory connectorFactory,
       CommandExceptionHandlingStrategy commandExceptionHandlingStrategy,
       SecretProviderAggregator secretProviderAggregator,
+      ValidationProvider validationProvider,
+      ObjectMapper objectMapper,
       MetricsRecorder metricsRecorder) {
     return new OutboundConnectorManager(
         jobWorkerManager,
         connectorFactory,
         commandExceptionHandlingStrategy,
         secretProviderAggregator,
+        validationProvider,
+        objectMapper,
         metricsRecorder);
```

Comment:
This constructor is getting bigger, which indicates that this class does to many things. I don't have a good proposal now how we can split/refactor the logic but we should definitely think about it. No need to take actions in this PR.